### ISSUE TITLE
Autocomplete form fields.


### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#479]
# Fields

Visibility: 3
Evaluation: Partially Supports
Success Criteria: 1.3.5: Identify Input Purpose
Needs design: 

### Description
Specify the autocomplete attribute and appropriate value where applicable to assist in form entry.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

# QA
Verify values are set as per:
- Create an account
   - [x] Email -> `email`
   - [x] Name -> `name`
   - [x] Password -> `new-password`
- Change account info
   - [x] Name -> `change-value`
   - [x] Email -> `change-value`
   - [x] Mobile -> `change-value`
- Add reply-to address
   - [x] Email -> none
- Sign in
   - [x] Email -> `username`
   - [x] Password -> `current-password`
- Send me a link (forgot pw)
   - [x]  Email -> `username`
- New stuff:
   - [x] Security code: ->  `one-time-code`
   - [x] Confirm account info change: ->  `current-password`
- Change password
   - [x] old password should be `current-password`
   - [x] new password should be `new-password`

**Comment by amazingphilippe** [comment]
We should review places where we ask for personal information. This is where autocomplete is useful. Otherwise we have a lot of forms where autocomplete can't provide any assistance.

**Comment by adriannelee** [comment]
Picking this up today

**Comment by adriannelee** [comment]
@amazingphilippe I think these might be it? https://miro.com/app/board/uXjVO7KLqH0=/?share_link_id=915377044756

**Comment by andrewleith** [comment]
Just went through the pages listed in the miro board, there is a mix of autocomplete already being on/off:
- Contact us (unauth)
   - Name -> on (autocomplete value is `name`)
   - Email -> on (`email`)
- Create an account
   - Email -> on (`email`)
   - Name -> not set  (chrome is autocompleting it anyway)
   - Password -> on (`new-password`)
- Change account info
   - Name -> not set
   - Email -> not set
   - Mobile -> not set
- Add reply-to address
   - Email -> not set
- Sign in
   - Email -> set to `off`
   - Password -> set to `off`
- Send me a link (forgot pw)
   - Email -> not set

@adriannelee, @amazingphilippe want to discuss what these values should be for each of these pages so we can move ahead with updating them?

**Comment by yaelberger-commits** [comment]
Phil, Adrianne and Andrew to sync or asynch

**Comment by andrewleith** [comment]
Ok as per our conversation, here is what we're aiming for:

- Create an account
   - Email -> on (email)
   - [ ] Name -> not set (chrome is autocompleting it anyway) :point_right: set to name
   - Password -> on (new-password)
- Change account info
   - Name -> change-name (off)
   - Email -> change-email (off)
   - Mobile -> change-tel (off)
- Add reply-to address
   - Email -> reply-to-email (off)
- Sign in
   - [ ] Email -> set to off :point_right: set to username
   - [ ] Password -> set to off :point_right: set to current-password
- Send me a link (forgot pw)
   - [ ]  Email -> not set :point_right: set to username
- New stuff:
   - [ ] Security code: :point_right: set to one-time-code
   - [ ] Confirm account info change: :point_right: set to current-password

**Comment by yaelberger-commits** [comment]
Release to staging, review then merge in prod

**Comment by sastels** [comment]
Most things in the QA list look good!

I don't see the autocomplete on the reply-to email
https://staging.notification.cdssandbox.xyz/services/d30c23bd-03ff-4c04-b5ee-202a78f921bd/service-settings/email-reply-to/add


```html
<input 
aria-describedby="email_address-hint" 
aria-required="true" 
class="form-control w-full input focus:shadow-outline" 
data-module="" 
id="email_address" 
name="email_address"
 required=""
 spellcheck="false"
 type="email" 
value=""
>
```

Also, where is the "Confirm account info change" - I can make changes to my profile without any confirmation.

**Comment by andrewleith** [comment]
Thanks @sastels 

> I don't see the autocomplete on the reply-to email https://staging.notification.cdssandbox.xyz/services/d30c23bd-03ff-4c04-b5ee-202a78f921bd/service-settings/email-reply-to/add

It's supposed to be omitted for this one, so we're good here!

> Also, where is the "Confirm account info change" - I can make changes to my profile without any confirmation.

Sorry, not very straight-forward.  This is at `Your account` -> `Your profile` -> `Email Adress` -> `Change`.
After you change your email it will ask you to confirm you password to save the change, its on that password field!

**Comment by sastels** [comment]
👍  Thanks for the info! All fields verified!
